packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -119,6 +154,38 @@ export class ProblemContribution extends AbstractViewContribution<ProblemWidget>
         }
     }
 
+    protected handleCopy(content: string): void {
+        const cp = document.createElement('textarea');
```

Comment:
Is it absolutely necessary to have to create and remove a `textarea` in the dom in order to copy?

Reply:
I believe there are many ways to implement "copying to clipboard" function, and this is one of them. Do you suggest doing in a more elegant way ;)?